<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
            </property>
            <property name="text">
             <string>Process Queue----</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView_process">
            <property name="styleSheet">
             <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_getReport">
          <property name="text">
           <string>Get report</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
            </property>
            <property name="text">
             <string>Run Recipe Details</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_recipeDetails">
            <property name="text">
             <string>Recipe Details</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_torquDerivedValues">
            <property name="text">
             <string>Torque &amp; Derived Values</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_timeValues">
            <property name="text">
             <string>Time Values (min)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_extendValues">
          <property name="text">
           <string>Extended Values</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_stop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_start">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
                </property>
                <property name="text">
                 <string>Parameters</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string>MI_Initail</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string>MI_Minimum</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string>MH_Maximum</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string>OptimumCare</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
                </property>
                <property name="text">
                 <string>Measured</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_16">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_17">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_16">
              <item>
               <widget class="QLabel" name="label_18">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
                </property>
                <property name="text">
                 <string>Parameter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_19">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string>Thermoplasticity</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_20">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string>CureRate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_21">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string>ReverseTime</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_22">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string>Trend</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_17">
              <item>
               <widget class="QLabel" name="label_23">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
                </property>
                <property name="text">
                 <string>Measured</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_24">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_25">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_26">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_27">
                <property name="styleSheet">
                 <string notr="true">border: 2px  solid lightblue; border-radius: 5px; background-color: lightblue</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_14" stretch="2,5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_temp"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_loadCell"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
          </property>
          <property name="text">
           <string>Time :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_time">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
          </property>
          <property name="text">
           <string>Torque:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_torque">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
          </property>
          <property name="text">
           <string>Tand:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_tand">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
          </property>
          <property name="text">
           <string>Upper:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_upper">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">border: 2px  solid lightblue; border-radius: 5px</string>
          </property>
          <property name="text">
           <string>Lower:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_lower">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_com">
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>COM inActive</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_comPort"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_calibration">
        <property name="text">
         <string>Calibration</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
